options.go

```diff
@@ -177,6 +183,25 @@ func IdentityResolverFunc(resolver IdentityResolver) Option {
 	}
 }
 
+// Installs RingChecksumStatPeriod, which determines the period between
```

Comment:
documentation nitpick: "Installs" is a weird word here in the context of the godoc (e.g. for a user). It might make more sense just to say _"RingChecksumStatPeriod determines the period between ..."_